pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>org.itsallcode</groupId>
     <artifactId>openfasttrace-maven-plugin</artifactId>
-    <version>1.2.1</version>
+    <version>1.3.0</version>
```

Comment:
@jakobbraun integration tests are failing after updating the version number. Do you have any idea why this happens?